src/field_theory/separable.lean

```diff
@@ -8,6 +8,7 @@ import algebra.polynomial.big_operators
 import field_theory.minimal_polynomial
 import field_theory.splitting_field
 import field_theory.tower
+import algebra.squarefree
```

Comment:
I made `algebra.squarefree` a pretty big import. It might be preferable to put this lemma on the file `algebra.squarefree` instead to accomodate that, idk.
(This is still fine with me, just wanted to make sure a maintainer checks this)

Reply:
Ah yes, that's also possible. Just let me know which one is better.

Reply:
I feel like this is the more natural location. Maybe we can minimize the imports in `algebra.squarefree` in a future PR.